girder/models/model_base.py

```diff
@@ -791,7 +918,7 @@ def setAccessList(self, doc, access, save=False):
 
         return doc
 
-    def setGroupAccess(self, doc, group, level, save=False):
+    def setGroupAccess(self, doc, group, level, save=False, flags=None, user=None):
```

Comment:
by the same logic that `setPublicFlags` has a `force` param, this method should too